packages/ketcher-macromolecules/src/components/modal/monomerConnection/MonomerConnections.tsx

```diff
@@ -200,6 +201,26 @@ function AttachmentPointSelectionPanel({
     const leavingGroup = MonomerCaps[attachmentPoint];
     return leavingGroup === 'O' ? 'OH' : (leavingGroup as LeavingGroup);
   };
+  let bond;
+  if (selectedAttachmentPoint) {
+    bond = monomer.getPotentialBond(selectedAttachmentPoint);
```

Comment:
maybe this could be included in `handleSelectAttachmentPoint ` method `if (selectedAttachmentPoint) {` block?